src/components/Registration/NaicsResult.vue

```diff
@@ -1,41 +1,95 @@
 <template>
-  <div class="naics-result">
-    <!-- *** TODO: format this as per comps -->
-    <v-btn text @click="emitClick(result)">
-      {{result.classTitle}}
-    </v-btn>
-  </div>
+  <v-row no-gutters class="naics-result" v-if="!!result" @click="emitClick(result)">
```

Comment:
If the user clicks on anything in this v-row (which doesn't handle and stop the click) then the click event bubbles up here. Hence, this whole div acts like a button.